### ISSUE TITLE
Assertion Failed shared_application.is_none and unimplemented function _CFURLCopyFileSystemPath for BOMBERMAN

### ISSUE THREAD

**Issue by TylerJaacks** [hikari-no-yume/touchHLE#194]
App bundle info:
- Display name: Bomberman
- Version: 1.0
- Identifier: jp.co.hudson.BombermanTouch
- Internal name (canonical): BombermanTouch.app
- Minimum OS version: (not specified)

`assertion failed: env.framework_state.uikit.ui_application.shared_application.is_none()`

Commenting out ui_application.rs:55 resolves the issue above so there must be something wrong related to UIApplication `-(id)init;`. However there is now a CoreFoundation function call `CFURLCopyFileSystemPath`.

Maybe this should be a separate issue. 

`Call to unimplemented function _CFURLCopyFileSystemPath`

**Comment by hikari-no-yume** [comment]
> Also, please bear in mind that there are infinitely many apps that do not work in touchHLE right now, so please don't open issues about broken apps unless one of these applies:
> 
> - You know that this specific version of the app worked in a previous version of touchHLE and you can reproduce this.
> - The app is partially working (e.g. loaded up to the menu but the main game doesn't work). The fact that an app's splash screen (Default.png) shows up doesn't mean it's partially working.
